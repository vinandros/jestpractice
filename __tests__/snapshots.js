const client = {
  name: "Kevin AndrÃ©s A",
  balance: 5000,
  type: "Premium",
};

describe("testing client", () => {
  test("should Kevin", () => {
    expect(client).toMatchSnapshot();
  });
});
