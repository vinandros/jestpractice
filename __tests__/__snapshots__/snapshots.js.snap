// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing client should Kevin 1`] = `
Object {
  "balance": 5000,
  "name": "Kevin Andr√©s A",
  "type": "Premium",
}
`;
